warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__schedule_feed_on_transitland.sql

```diff
@@ -0,0 +1,38 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('stg_gtfs_guidelines__feed_guideline_index') }}
+    WHERE check = {{ schedule_feed_on_transitland() }}
+),
+
+fact_daily_transitland_url_check AS (
+SELECT * FROM {{ ref('stg_gtfs_guidelines__fact_daily_transitland_url_check') }}
+),
+
+schedule_daily_check AS (
```

Comment:
(this comment applies to all 4 checks) I think that you need to do some deduplication here (either `SELECT DISTINCT` or a group by or something else) -- I think that you might get fanout on this because (with the url number issue) this table can have multiple rows for one ITP ID that meet this condition.

(I am going to try to be more mindful of this, now that I've noticed some of the failing tests I messaged you about -- I suspect there are other checks that may have a version of this issue.)

Reply:
ah right, of course. yes I will dedupe here.